### PR TITLE
Updating packager and assembly versions for servicing.

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
Getting the branch ready for servicing.

### CODE REVIEW

File: pkg/baseline/packageIndex.json [hunk 1]
```diff
@@ -22,11 +22,12 @@
         "4.3.0",
         "4.4.0"
       ],
-      "BaselineVersion": "4.4.0",
```
Comment (weshaggard) [inline pkg/baseline/packageIndex.json]: You shouldn't update the baselineversions.

File: Packaging.props [hunk 2]
```diff
@@ -20,7 +20,7 @@
     <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">1.2.0</PlatformPackageVersion>
 
     <!-- by default all packages will use the same version which revs with respect to product version -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">4.4.0</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">4.4.1</PackageVersion>
```
Comment (weshaggard) [inline Packaging.props]: Do you really want to update all packages? I would suggest just updating the one you want to ship and not build the others.

File: src/System.ServiceModel.Primitives/dir.props [hunk 3]
```diff
@@ -2,6 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.2.0.0</AssemblyVersion>
+    <AssemblyVersion>4.2.0.1</AssemblyVersion>
```
Comment (weshaggard) [inline src/System.ServiceModel.Primitives/dir.props]: If you aren't changing anythink you don't really need to update the versions.

File: Packaging.props (hunk 2 above)
Comment (weshaggard) [reply Packaging.props]: That is what we do in corefx. We only add the packages we want to ship to the packages.builds file https://github.com/dotnet/corefx/blob/release/2.0.0/src/packages.builds#L19.

Comment (zhenlan) [reply Packaging.props]: I think we should respin all WCF packages because people may not use the S.SM.Primitives package directly.

Comment (weshaggard) [reply Packaging.props]: Of course that is your choice.

**Comment by StephenBonikowsky** [comment]
After speaking with @zhenlan will amend this PR shortly.

File: Packaging.props (hunk 2 above)
Comment (zhenlan) [reply Packaging.props]: After discussed with @weshaggard offline, I think we can go with the plan to rebuild System.ServiceModel.Primitives only as the compat pack will be updated to reference the newest version of this package. The compat pack is what customers will use in the end.

**Comment by StephenBonikowsky** [comment]
This now only builds a new package of System.ServiceModel.Primitives but same assembly version since nothing changed.

**Comment by StephenBonikowsky** [comment]
Just realized I need to figure out how to make the use a fully stable version, currently it is...
System.ServiceModel.Primitives.4.4.1-servicing-25910-0.nupkg

**Comment by weshaggard** [comment]
>Just realized I need to figure out how to make the use a fully stable version, currently it is...
System.ServiceModel.Primitives.4.4.1-servicing-25910-0.nupkg

That is what you want until you are ready to stabilize it. I would suggest get a least one official build before you do that.

Once you do that to stabilize just add the version https://github.com/dotnet/wcf/pull/2371/files#diff-c5acbfc4baa444e6a7321d2a8b07ccbaL85.

**Comment by StephenBonikowsky** [comment]
Ok, will stabilize when ready for an actual servicing release.

**Pull request merged**